python/tvm/build_module.py

```diff
@@ -4,13 +4,16 @@
 LoweredFunc and compiled Module.
 """
 from __future__ import absolute_import as _abs
+from __future__ import print_function
```

Comment:
do not use print for file write, always use f.write instead

Reply:
Ok

Reply:
seems f.write doesn't support stmt;
 File "test_pass_unroll.py", line 19, in test_unroll_loop
    f.write(stmt)
TypeError: expected a string or other character buffer object

Reply:
f.write(str(stmt))